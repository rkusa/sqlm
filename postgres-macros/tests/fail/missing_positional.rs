fn main() {
    sqlm_postgres_macros::sql!("foo{1} {}", "bar");
}
