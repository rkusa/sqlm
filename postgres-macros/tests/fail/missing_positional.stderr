error: missing argument for index 1
 --> tests/fail/missing_positional.rs:2:32
  |
2 |     sqlm_postgres_macros::sql!("foo{1} {}", "bar");
  |                                ^^^^^^^^^^^
